app/models/thing.rb

```diff
@@ -6,7 +6,7 @@ class Thing < ActiveRecord::Base
   extend Forwardable
   include ActiveModel::ForbiddenAttributesProtection
 
-  VALID_DRAIN_TYPES = ['Storm Water Inlet Drain', 'Catch Basin Drain'].freeze
+  VALID_DRAIN_TYPES = ['Storm Drain', 'Storm Water Inlet Drain', 'Catch Basin Drain'].freeze
```

Comment:
I expect this to change again.  Do we need a test here or should we an assertion in the import process?